sdks/python/setup.py

```diff
@@ -102,16 +102,17 @@ def get_version():
     cythonize = lambda *args, **kwargs: []
 
 REQUIRED_PACKAGES_PY2_ONLY = [
-    'avro>=1.8.1,<2.0.0'
+    'avro>=1.8.1,<2.0.0',
+    'dill>=0.2.6,<=0.2.8.2',
 ]
 
 REQUIRED_PACKAGES_PY3_ONLY = [
-    'avro-python3>=1.8.1,<2.0.0'
+    'avro-python3>=1.8.1,<2.0.0',
```

Comment:
We can simplify requirements spec  with `python_version` qualifier: See example at: https://github.com/apache/beam/blob/95d0ac5e5cb59fd0c6a8a4861a38a7087a6c46b5/sdks/python/container/base_image_requirements.txt#L27

Reply:
This doesn't seem to work for the dill requirement. We can clean this up when we clean up the dill dependency.